backend/tests/pytest.ini

```diff
@@ -0,0 +1,5 @@
+[pytest]
```

Comment:
Since we already use pyproject.toml, can we just add this there?

https://docs.pytest.org/en/stable/reference/customize.html

Reply:
Moved